### PR TITLE
Add 'linter-swiftlint.additionalOptions' configuration

### PR DESCRIPTION
**Pull request by norio-nomura**

### CODE REVIEW

File: lib/init.coffee [hunk 1]
```diff
@@ -4,6 +4,11 @@ module.exports =
       type: 'string'
       title: 'the path to SwiftLint\'s configuration file that relative to project or absolute'
       default: '.swiftlint.yml'
+    additionalOptions:
+      type: 'string'
+      title: 'additional options to SwiftLint\'s `lint` command
```
Comment (Arcanemagus) [inline lib/init.coffee]: This should be a `description`, not `title` (yes, all the other options are wrong... 😛).

Comment (Arcanemagus) [reply lib/init.coffee]: Also, remove the newline from the middle of the string (eg. by concatenation)

**Review by Arcanemagus — CHANGES_REQUESTED** [review]

**Comment by Arcanemagus** [comment]
@keplersj Any objection to adding @norio-nomura as a collaborator on here?

Assuming that is something you would want @norio-nomura 😉

**Comment by keplersj** [comment]
@Arcanemagus I think that's a very good idea and I've offered it to @norio-nomura [before](https://github.com/AtomLinter/linter-swiftlint/pull/8#issuecomment-172714637).

**Comment by Arcanemagus** [comment]
Ah, good to know lol.

**Comment by norio-nomura** [comment]
Thank you for offering again.
I didn't think I want to be a collaborator on here. Even though my way of contribution will not change with that, I should accept that as long as I look at `git blame`.

File: lib/init.coffee [hunk 2]
```diff
@@ -2,11 +2,17 @@ module.exports =
   config:
     configurationPath:
       type: 'string'
-      title: 'the path to SwiftLint\'s configuration file that relative to project or absolute'
+      title: 'Configuration Path'
```
Comment (Arcanemagus) [inline lib/init.coffee]: Atom should default to this title, is it not?

File: lib/init.coffee [hunk 3]
```diff
@@ -2,11 +2,17 @@ module.exports =
...
+      description: 'SwiftLint\'s configuration file that relative to project or absolute'
       default: '.swiftlint.yml'
+    additionalOptions:
+      type: 'string'
+      title: 'Additional Options'
```
Comment (Arcanemagus) [inline lib/init.coffee]: Same with this one. Atom generates a title based on the option name, capitalizing the first letter, and inserting a space whenever it finds a capital letter. So `additionalOptions` _should_ automatically be being translated into `Additional Options`.

**Review by Arcanemagus — CHANGES_REQUESTED** [review]

**Review by Arcanemagus — APPROVED** [review]
LGTM

**Comment by Arcanemagus** [comment]
This works as is as far as I'm concerned, the only thing I can think about as far as this PR is concerned is adding a check to make sure the user hasn't specified `--config` in there.

Good to merge @keplersj?

**Comment by norio-nomura** [comment]
Thanks! 🙏

**Comment by Arcanemagus** [comment]
Published in v1.2.0. 🎉

**Pull request merged**